assessment.js

```diff
@@ -0,0 +1,112 @@
+document.addEventListener("DOMContentLoaded", () => {
+    console.log("DOMContent is loaded")
+    selectTitle()
+    // displaySelectedFilm(film)                                                                                                                                                                                      select", selectTitle)
+    // let reviewInput = document.querySelector("#text-input")
+    // let form = document.querySelector("form")
+    // form.addEventListener("submit", (event) => {
+    //       let textInput = document.querySelector("#text-input")
+    //textInput.innerText = ""
+    let inputBtn = document.querySelector("#btn-input")
+    inputBtn.addEventListener("submit", (reviewSection))
+    //event.preventDefault()
+    //   if (textInput.innerText = "") {
+    //      form.appendChild(textInput)
+    // })
+
+    // console.log("review box", reviewInput)
+    // reviewInput.addEventListener("click",  {
+
+    // })
+})
+
+const selectTitle = () => {
```

Comment:
Function names do not fully convey what the function does, making it hard to understand without carefully looking at the code. For instance `selectedTitle()` is fetching/getting all the movies form the API, a better name would have been `getAllMovies`.

Reply:
You did a good on `selectTitle()`, despite the inaccurate name the function does one thing: fetch the movies. It then calls another function to take care of displaying the info. This is what we want one function doing one thing, then having multiple functions that when put together accomplish a greater task.